linux/LinuxMDEparser/README.MD

```diff
@@ -0,0 +1,51 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux.
+
+# About
+The LinuxMDEparser is a tool that allows you to export or parse MDE for Linux log files to Excel, making it easier to filter and search the logs. After collecting the MDE logs from any Linux machine, you can then run this tool on your on Windows.
+
+# Usage
+python3 main.py {Command} <option>
+
+EX: python3 main.py real-time-protection --convert
+    python3 main.py wdavhistory -h
+
+Make sure you run it from the folder where the log file is located. Excel file will also be created in the same folder.
+
+Files are included when you collect the Linux - "mdatp diagnostic create" command in Linux, except for real_time_protection.json - this ones need to be collected locally in a terminal window:
+
+mdatp diagnostic real-time-protection-statistics --output json > real_time_protection.json
+
+More details https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/linux-support-perf
+
+# Help
+You can always check the available options by running: LinuxMDEparser -h
+
+usage: LinuxMDEparser.exe [command] [--option]
```

Comment:
Still .exe

Reply:
This is the help from the python and I rather prefer leave it like this. I can remove the help output if you prefer ...

Reply:
@bmrodrigues , the help output will be without .exe on all platforms but Windows (and I believe that macOS/Linux is our primary platform for this repository). I know that many non-Windows users will get confused (and will open issues agains our code) if they see .exe and do not know where to get it. On the other hands Windows users will most probably ignore a missed .exe - as it is common to run executables on Windows omitting .exe